content/pages/disability-benefits/after-you-apply.md

```diff
@@ -57,8 +57,11 @@ You don’t need to do anything unless we send you a letter asking for more info
 <li class="process-step list-one">
 
 #### Claim Received
+We’ll let you know when we receive your claim. Depending on where you filed your disability claim, you’ll either get a message from us on eBenefits or by email. 
```

Comment:
Should we reverse "claim" and "disability claim" here, so that we have specified which claim first, and then use a less specific expression afterwards? So like:
"We’ll let you know when we receive your disability claim. Depending on where you filed your claim..."